zad1/main.rb

```diff
@@ -1,21 +1,15 @@
 def sort_letters(string)
-
+	string.gsub(/\s+/, '').split('').sort.join
 end
 
 def count_vowels(string)
-
+	def isVowel s
+		'aeiouy'.each_char {|a| return true if a.eql?(s)}
+	false
 end
-
-def filter_odd(array)
-
+	string.each_char.select {|a| isVowel(a)}.length
```

Comment:
Ogółem w Ruby nie praktykuje się definiowania metod w metodach, a  isVowel(Btw w Ruby używa się snake_case) można usunąć przez użycie metody [count](https://ruby-doc.org/core-2.5.3/Enumerable.html#method-i-count) na zwróconym arrayu. Można zrobić to oczywiście jeszcze prościej używając metody `count`, która występuje analogicznie w klasie [String](https://ruby-doc.org/core-2.2.0/String.html#method-i-count)

Reply:
Właśnie zauważyłem to rozwiązanie u innych osób po wysłaniu swojego, a przy przeglądaniu dostępnych metod jakoś przeoczyłem tę konkretną metodę :)

Reply:
![ruby_you_can_but_dont](https://user-images.githubusercontent.com/18199403/47956354-80e5b080-dfa3-11e8-859c-79cca8529550.png)
😂 😂